spartan/terraform/deploy-aztec-infra/values/prover.yaml

```diff
@@ -1,7 +1,8 @@
 node:
   node:
     env:
-      L1_FIXED_PRIORITY_FEE_PER_GAS: 3
+      L1_PRIORITY_FEE_BUMP_PERCENTAGE: 30
+      L1_PRIORITY_FEE_RETRY_BUMP_PERCENTAGE: 80
```

Comment:
I don't understand why here these are set as env vars and in the previous ones as yaml values, and at this point I'm too afraid to ask.